coderd/provisionerdserver/provisionertags.go

```diff
@@ -24,7 +24,9 @@ func MutateTags(userID uuid.UUID, tags map[string]string) map[string]string {
 	}
 	switch tags[TagScope] {
 	case ScopeUser:
-		tags[TagOwner] = userID.String()
+		if userID != uuid.Nil {
+			tags[TagOwner] = userID.String()
+		}
```

Comment:
I'd like to see an `else` branch that unsets the owner, or alternatively, don't call this function for PSK logins and just directly set `scope=organization`.

I don't think we want to allow external provisioners with the PSK to claim builds meant for people local workstations or whatever.

Reply:
I think the latter makes more sense. Provisioner daemons authenticating with PSKs are implicitly organisation-scoped anyway.